flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/exceptionhistory/TestAccessExecutionBuilder.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.scheduler.exceptionhistory;
+
+import org.apache.flink.runtime.execution.ExecutionState;
+import org.apache.flink.runtime.executiongraph.ErrorInfo;
+import org.apache.flink.runtime.executiongraph.ExecutionAttemptID;
+import org.apache.flink.runtime.taskmanager.LocalTaskManagerLocation;
+import org.apache.flink.runtime.taskmanager.TaskManagerLocation;
+
+/** Builder for a {@link TestAccessExecution}. */
+public class TestAccessExecutionBuilder {
+    private ExecutionState state = null;
+    private ErrorInfo failureInfo = null;
+    private ExecutionAttemptID attemptId = new ExecutionAttemptID();
+    private TaskManagerLocation taskManagerLocation = new LocalTaskManagerLocation();
+
+    public TestAccessExecutionBuilder withAttemptId(ExecutionAttemptID attemptId) {
```

Comment:
Other builders are mostly using the `set...` prefix, it would be great to stick to that